datayoga/blocks/expression.py

```diff
@@ -32,12 +32,58 @@ def search(self, data: Any) -> Any:
         """
         pass
 
+    def test(self, data: Any) -> bool:
+        """Test a where clause for an SQL statement
+
+        Args:
+            data (Any): Data
+
+        Returns:
+            boolean: True if matches, else False
+        """
+        pass
+
 
 class SQLExpression(Expression):
     def compile(self, expression: str):
         self.conn = sqlite3.connect(":memory")
         self.expression = expression
 
+    def filter(self, data: List[Any]) -> List[Any]:
```

Comment:
should be changed to List[Dict[str, Any]] (both places)

Reply:
done